src/constants/organisations/organisationTypes.ts

```diff
@@ -5,19 +5,15 @@ export enum OrganisationTypeEnum {
   company = 'company',
   cooperative = 'cooperative',
   groupOfFriends = 'groupOfFriends',
-  other = 'other',
   publicOrRegionalAuthority = 'publicOrRegionalAuthority',
   universityOrSchool = 'universityOrSchool',
+  other = 'other',
 }
 
 export const ORGANISATION_TYPES = {
   [OrganisationTypeEnum.company]: t('Entreprise'),
-  [OrganisationTypeEnum.publicOrRegionalAuthority]: t(
-    'Public ou collectivité territoriale'
-  ),
-  [OrganisationTypeEnum.cooperative]: t('Coopérative'),
   [OrganisationTypeEnum.association]: t('Association'),
+  [OrganisationTypeEnum.publicOrRegionalAuthority]: t('Service public'),
   [OrganisationTypeEnum.universityOrSchool]: t('Université ou école'),
-  [OrganisationTypeEnum.groupOfFriends]: t("Groupe d'amis"),
   [OrganisationTypeEnum.other]: t('Autre'),
```

Comment:
You need to keep the 7 values here otherwise not translated

Reply:
okay